layouts/partials/footer.html

```diff
@@ -1,15 +1,13 @@
-{{ $logo := printf "img/logos/%s" site.Params.logos.footer | relURL }}
-{{ $menu := site.Menus.main }}
```

Comment:
Single-use variable. We don't loose clarity be inlining. Have a local variable declaration only makes me wonder if there are multiple uses of the variable.